docs/userguide.rst

```diff
@@ -93,6 +93,8 @@ For detailed information, see the API reference of the associated class.
 +--------------+-----------------------------------------------------------------------------+-------------------------------------+----------------------------------------------------+
 | file         | ``file:/tmp/file-to-push``                                                  | n/a                                 | Obtain a single file-backed item of various types. |
 +--------------+-----------------------------------------------------------------------------+                                     |                                                    |
+| cgw         | ``cgw:/tmp/yaml-file-to-push``                                               |                                     |                                                    |
```

Comment:
I think there is a whitespace error here?  Doesn't seem to line up perfectly, although it's not easy to tell in the github UI.

```suggestion
| cgw          | ``cgw:/tmp/yaml-file-to-push``                                               |                                     |                                                    |
```